de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ProjectUiAllTests.java

```diff
@@ -26,9 +30,11 @@
 	EditorTest.class,
 	NewProjectWizardTest.class,
 	CutCopyDeleteUndoTest.class,
-	InheritanceTest.class, 
-	CalculationTest.class,   
+	InheritanceTest.class,
+	CalculationTest.class,
 	ProductStructureTest.class,
+	GitVersioningBackendAndUserRightsManagementTest.class,
+	SvnVersioningBackendAndUserRightsManagementTest.class,

```

Comment:
Should we rename these classes and the abstract class, because we added a seperate class for RoleManagementTest in https://github.com/virtualsatellite/VirtualSatellite4-Core/issues/649 ? E.g. just GitVersioningBackendTest